### PR TITLE
[bug fix]fix TaskManager initialDelay not correct

### PR DESCRIPTION
**Pull request by zddr**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

when timeunit is not second,delay  will not correct

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/common/util/TimeUtils.java [hunk 1]
```diff
@@ -269,4 +269,25 @@ public static TimeUnit convertUnitIdentifierToTimeUnit(String unitIdentifierDesc
                         "Can not get TimeUnit from UnitIdentifier description: " + unitIdentifierDescription);
         }
     }
+
+    public static long convertSecondToTimeUnitValue(long second, TimeUnit unit) {
```
Comment (Astralidea) [inline fe/fe-core/src/main/java/com/starrocks/common/util/TimeUtils.java:273]: I think maybe we do not need convert. what about unify to second?

**Review by Astralidea — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 8 / 16 (50.00%)




#### file detail

|   |path|covered line|new line|coverage|not covered line detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/TaskManager.java|0|7|00.00%|[129, 130, 131, 350, 379, 544, 545]|
|:large_blue_circle:|com/starrocks/common/util/TimeUtils.java|8|9|88.89%|[290]|

**Comment by Astralidea** [comment]
run starrocks_admit_test

**Comment by Astralidea** [comment]
run starrocks_admit_test

**Review by kangkaisen — APPROVED** [review]

**Comment by kangkaisen** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by Astralidea** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by zddr** [comment]
run starrocks_fe_unittest

**Comment by Astralidea** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 8 / 16 (50.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/TaskManager.java|0|7|00.00%|[129, 130, 131, 350, 379, 544, 545]|
|:large_blue_circle:|com/starrocks/common/util/TimeUtils.java|8|9|88.89%|[290]|

**Comment by Astralidea** [comment]
run starrocks_admit_test

**Pull request merged**